actions/crib-deploy-environment/action.yml

```diff
@@ -75,6 +75,13 @@ outputs:
   devspace-namespace:
     description: "Kubernetes namespace used to provision a CRIB environment."
     value: ${{ steps.generate-ns-name.outputs.devspace-namespace }}
+  ingress-service-names-env:
+    description:
+      "Aggregated key-value pairs of Ingress service endpoints in .env format,
+      excluding the ingress base domain. To construct the fully qualified domain
+      names (FQDNs) for accessing the services, append the base domain provided
+      as the ingress-base-domain input."
+    value: ${{ steps.get-ingress-services.outputs.ingress-service-names-env }}
```

Comment:
It's probably ok if we assume that this will be consumed by a shell script, but in general I would prefer more universal format for output like JSON. The consumer could read the variable by `echo $ingressess | jq .INGRESS_NAME`.

Reply:
Thanks. It should be all set.